vividus-plugin-ssh/src/main/resources/spring.xml

```diff
@@ -11,15 +11,17 @@
 
     <context:component-scan base-package="org.vividus.ssh" />
 
-    <bean class="org.vividus.context.SshTestContext" />
+    <bean class="org.vividus.ssh.converter.CommandsConverter" />
```

Comment:
What is the reason for this change?

Reply:
general change in order to keep everything under `org.vividus.ssh` package